docs/actions.md

```diff
@@ -1219,6 +1184,71 @@ This command starts a polling loop that continuously checks for logs from activa
 
   Similarly, whenever you run the poll utility, you see in real time the logs for any actions running on your behalf in OpenWhisk.
 
+## Getting actions
+
+Metadata that describes existing actions can be retrieved via the `wsk action get` command.
+
+```
+wsk action get hello
+ok: got action hello
+{
+    "namespace": "user@email.com",
+    "name": "hello",
+    "version": "0.0.1",
+    "exec": {
+        "kind": "nodejs:6",
+        "binary": false
+    },
+    "annotations": [
+        {
+            "key": "exec",
+            "value": "nodejs:6"
+        }
+    ],
+    "limits": {
+        "timeout": 60000,
+        "memory": 256,
+        "logs": 10
+    },
+    "publish": false
+}
+```
+
+### Getting an action URL
+
+An action can be invoked through the REST interface via an HTTPS request. To get an action URL, execute the following command:
+
+```
+wsk action get actionName --url
+```
+```
+ok: got action actionName
+https://${APIHOST}/api/v1/namespaces/${NAMESPACE}/actions/actionName
+```
+
+**Note:** Authentication must be provided when invoking an action via an HTTPS request. For more information regarding
```

Comment:
if the action is a web action, authentication is not required...   not sure how/if web action caveats need to be in this doc..

Reply:
I think a caveat is warranted.

Reply:
Hmm, I don't think we need a caveat here. There is an example in the web actions doc on how to use the `--url` flag. Also the doc says `action` not `web action`.

Reply:
I added the caveat just to be clear about how it works.